shopify/wishlist-public-remix-ssr/.template/config.json

```diff
@@ -0,0 +1,75 @@
+{
+  "version": "v1",
+  "setup": "./setup.md",
+  "webflow": {
+    "title": "Product Wishlists",
+    "description": "A complete wishlist solution for Shopify. Allows customers to create and manage multiple wishlists, and integrates seamlessly into the storefront and customer accounts.",
+    "content": "webflow/content.md",
+    "mainImage": "webflow/wishlist.webp",
+    "features": {
+      "shopify": true,
+      "frontend": true,
+      "models": true,
+      "actions": true
+    }
+  },
+  "featuredModels": {
+    "wishlist": {
+      "description": "Stores a named wishlist and associates it with a specific customer and shop.",
+      "featuredFields": ["name", "customer", "shop"]
+    },
+    "wishlistItem": {
+      "description": "Acts as a join table, linking a specific product variant to a wishlist.",
+      "featuredFields": ["variant", "wishlist", "customer"]
+    },
+    "shopifyCustomer": {
+      "description": "Extends the standard shopifyCustomer model to include relationships to all wishlists and wishlist items created by that customer.",
+      "featuredFields": ["wishlists", "wishlistItems"]
+    }
+  },
```

Comment:
It appears that you have added a `defaultUpdateFrequency` field to the `shopifyShop` model. What does it do? You should mention it here.

Reply:
Yessir